mlir/lib/Target/Cpp/TranslateToCpp.cpp

```diff
@@ -1129,6 +1171,50 @@ static LogicalResult printOperation(CppEmitter &emitter,
   return success();
 }
 
+static LogicalResult emitClassFields(CppEmitter &emitter,
+                                     emitc::FuncOp functionOp) {
+  raw_indented_ostream &os = emitter.ostream();
+  auto argAttrs = functionOp.getArgAttrs();
+  Operation *operation = functionOp.getOperation();
+  if (failed(printFields(emitter, operation, functionOp.getArguments())))
+    return failure();
+  os << ";\n";
+
+  std::map<std::string, Value> fields;
+  os << "std::map<std::string, char*> _buffer_map {";
+  if (argAttrs) {
+    bool isFirst = true;
```

Comment:
nit: I think we can always add a `,` after an element in the dictionary initializer list.